src/overlays/actors/ovl_En_Fishing/z_en_fishing.c

```diff
@@ -2590,8 +2583,7 @@ void EnFishing_UpdateLure(EnFishing* this, GlobalContext* globalCtx) {
                     EnFishing_SpawnRipple(NULL, globalCtx->specialEffects, &spA8, 30.0f, 300.0f, 150, 90);
                 }
             }
-            break;
-        }
+        } break; // must be outside of the block
```

Comment:
I think the upshot was this can go on a new line.

Reply:
Oh right, I cleaned everything else up from the scratch but forgot this, thanks.

Reply:
Oh, it turns out that if you put this on a new line, then run` ./format.sh`, it forces it back to how it is here. I think that's fine personally, but I'm letting you know just in case we want to override the formatter.

Reply:
Nah, I guess that's some weird feature we've never had to deal with before.